ImageClassification/ModelDataHandler/ModelDataHandler.swift

```diff
@@ -227,18 +229,18 @@ class ModelDataHandler {
   ///       floating point values).
   /// - Returns: The RGB data representation of the image buffer or `nil` if the buffer could not be
   ///     converted.
-  private func rgbDataFromBuffer(
-    _ buffer: CVPixelBuffer,
-    byteCount: Int,
-    isModelQuantized: Bool
-  ) -> Data? {
-    CVPixelBufferLockBaseAddress(buffer, .readOnly)
-    defer {
-      CVPixelBufferUnlockBaseAddress(buffer, .readOnly)
-    }
-    guard let sourceData = CVPixelBufferGetBaseAddress(buffer) else {
-      return nil
-    }
+	private func rgbDataFromBuffer(
+		_ buffer: CVPixelBuffer,
+		byteCount: Int,
+		isModelQuantized: Bool
+	) -> Data? {
```

Comment:
I think I'm going to refactor all the preprocessing into separate functions, and some how separate the video classifier code with the internal model code.